### DISCUSSION TITLE
Go Prisma is working great so far!

### DISCUSSION THREAD

**Discussion by dangdennis** [prisma/prisma-client-go#891] [Show and tell]
We at Yomi Games started a new web service with gin and prisma. Your docs and APIs were great to get going. We already use Prisma with Node on a different project and want to share the same database + schema with this service. We're using cockroachdb. 

I was hoping to see this project alive, and here it is, revived! 

If we run into issues, we'll make some pull requests. Our usage for this service is simple so far: gets (no joins), updates. 

A thing I'd like to see are helper methods to extract sql conflict errors. Sql conflict errors come back as raw strings or something. I didn't attempt to parse them for the right fields. The js prisma client doesn't offer this API either though. For now, I've ignored the conflict and made another sql query just to check. Would be nice to be able to do `errors.is(err, db.ErrorConflict)`.

**Comment by steebchen** [comment]
Thank you for the feedback, I'm very happy to hear this!

Regarding the errors, there is #213 to provide proper error handling which would include sql conflicts (aka constraint errors), such as if you insert a new row but the ID already exists. I want to do this soonish, but have some other things to fix before that.

If you depend on the Go client long term and want to see faster development, I'd be incredibly grateful if you would [sponsor me](https://github.com/sponsors/steebchen) for any amount without any lock-in. However, that is of course completely optional, and contributing by spreading the word or just reporting bugs / discussing features is also very helpful, so please let me know any feedback!

Feel free to join the community discord: https://discord.gg/g6Vk2z6d
